pkg/kubelet/kubelet.go

```diff
@@ -1468,8 +1468,10 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 	go kl.volumeManager.Run(kl.sourcesReady, wait.NeverStop)
 
 	if kl.kubeClient != nil {
-		// Start syncing node status immediately, this may set up things the runtime needs to run.
-		go wait.Until(kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, wait.NeverStop)
+		// Introduce some small jittering to ensure that over time the requests won't start
+		// accumulating at approximately the same time from the set of nodes due to priority and
+		// fairness effect.
+		go wait.JitterUntil(kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, 0.04, true, wait.NeverStop)
```

Comment:
To follow up on the 0.04 vs 0.1:
- withe default 10s statusUpdateFrequency, 0.1 means 1s, so we would be calculating every 9s-11s. That sounded acceptable to me.
- that said with 0.04 (so 9.6s-10.4s with default frequency) - we seem to be getting good enough spread over time for extended periods of time.
So I'm completely fine with setting it to 0.04 for consistency. If at some point we realize we need more - we can revisit it, but it doesn't seem to be the case now.